app/views/dashboard/work_versions/_show_files.html.erb

```diff
@@ -0,0 +1,20 @@
+<% if work_version.file_resources.present? %>
```

Comment:
@rschenk You could move the if statement into show.html.erb: `render 'show_files' [...] if work_version.file_resources.present?` and eliminate the nesting.